### ISSUE TITLE
venv problem

### ISSUE THREAD

**Issue by LI-CHIN-CHIA** [mueller-franzes/medfusion#17]
Hi, I have some questions about the environment.
In the requirements.txt file, I want to know the versions of these packages. I encountered version problems when executing the program.
torch
numpy
scikit-learn
pytorch-lightning
pytorch_msssim
monai
torchmetrics
torch-fidelity
torchio
pillow
einops
torchvision
matplotlib
pandas
lpips
streamlit
python

**Comment by taranrai** [comment]
Yes, this would be super helpful as I'm encountering issues with versions. I believe the later pytorch_lightening is causing me issues with some imports.

**Comment by fawad1997** [comment]
Yes, please provide the versions along with the dependencies. Thanks
